source/geometry/FloatTransform3D.ooc

```diff
@@ -244,16 +244,16 @@ FloatTransform3D: cover {
 		This new(focalLength, 0, 0, 0, 0, focalLength, 0, 0, 0, 0, focalLength, 1.0f, 0, 0, 0, 0)
 	}
 	referenceToNormalized: func ~float (imageSize: FloatVector2D) -> This {
-		toReference := This createScaling(imageSize x / 2.0f, imageSize y / 2.0f, 1.0f)
-		toNormalized := This createScaling(2.0f / imageSize x, 2.0f / imageSize y, 1.0f)
+		toReference := This createScaling(imageSize x / 2.0f, imageSize y / 2.0f, imageSize x / 2.0f)
```

Comment:
Because when you apply uniform scaling on the resolution and the transform affects depth, you want to preserve the rotation along X and Y so that 45 degrees to the right remains 45 degrees to the right no matter the resolution of the image on the plane. I chose X because I will soon define normalized focal length relative to the width of the image.